tests/Jellyfin.Naming.Tests/Video/VideoResolverTests.cs

```diff
@@ -1,275 +1,205 @@
-﻿using MediaBrowser.Model.Entities;
+﻿using System.Collections;
+using System.Collections.Generic;
+using Emby.Naming.Common;
+using Emby.Naming.Video;
+using MediaBrowser.Model.Entities;
 using Xunit;
 
 namespace Jellyfin.Naming.Tests.Video
 {
     public class VideoResolverTests : BaseVideoTest
```

Comment:
I think you forgot to remove `BaseVideoTest` here? I think that base class can also be deleted if you remove it from here

Reply:
I left it in, didn't want the diff to be too large

Reply:
Is `BaseVideoTest` unused now though? I thought it would just add that one deleted file to the diff, or is there more to it?

Reply:
Didn't notice it was unused, you are correct